packages/@guardian/source-react-components/src/icons/SvgPinned.tsx

```diff
@@ -0,0 +1,17 @@
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { iconSize } from '@guardian/source-foundations';
+import type { IconProps } from './types';
+
+export const SvgPinned = ({ size }: IconProps): EmotionJSX.Element => (
+	<svg
+		viewBox="0 0 24 24"
```

Comment:
Can we apply the @mxdvl hack to make the viewbox 30x30, in keeping with the other icons in Source?

```suggestion
		viewBox="-3 -3 30 30"
```

Reply:
Absolutely! All updated and commits squashed :)

Reply:
I'll have a look at moving all the icons to a new format on 10% day